### PR TITLE
react 1.10 rns updated

### PR DESCRIPTION
**Pull request by jenniferopal**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [X] Maintenance/Chore

### CODE REVIEW

**Review by Andrew-Hayes — DISMISSED** [review]
LGTM

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 ### React SDK
 
+#### Version 1.10.0
+
+ - We've updated the Javascript SDK to 1.26.0.
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:66]: I noticed that 1.9.0 below calls out the change to the Javascript Client SDK explictly - might want to do the same here?

Comment (jenniferopal) [reply release-notes/feature-flags.md:66]: Noted. I'll changed that.

Comment (erdirowlands) [reply release-notes/feature-flags.md]: @jenniferopal sorry I meant the actual changes made to the JS SDK version - e.g. " We've updated the JS SDK to x which does the following"

Comment (jenniferopal) [reply release-notes/feature-flags.md]: Ohhhh, I misread what you said. My mistake, I'll make the change now. :)

**Review by erdirowlands — APPROVED** [review]

**Pull request merged**